manifest_staging/charts/gatekeeper/values.yaml

```diff
@@ -11,6 +10,14 @@ logLevel: INFO
 logDenies: false
 emitAdmissionEvents: false
 emitAuditEvents: false
+postInstall:
+  labelNamespace:
+    enabled: false
+    image:
+      repository: bskim45/helm-kubectl-jq
```

Comment:
is this only used for `kubectl`? can we re-use `line/kubectl-kustomize` image we use for kustomize?

Reply:
yep for sure will look at using that.